app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,54 @@
+class Admin::UsersController < ApplicationController
+  before_action :find_user, only: [:edit, :update, :destroy, :show]
+  before_action :admin_required
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to admin_user_path(current_user.id)
```

Comment:
下記の形で id を指定しなくてもオブジェクト自体を渡すことでURL生成は行えるのでご確認ください。
```suggestion
      redirect_to admin_user_path(current_user)
```

Reply:
ありがとうございます。テストが通ることも確認いたしました。
対応箇所の修正を行います。

Reply:
7f437df
e648ce4
8816c4c
で修正いたしました。